lms/static/sass/elements/_vendor.scss

```diff
@@ -0,0 +1,202 @@
+    border-color: $mediumGrey;
+    color: $blue;
+    @include transition(none);
+
+    &.ui-state-hover {
+      background: $lightBluishGrey;
+      border-color: $lightBluishGrey;
+      color: $white;
+    }
+  }
+
+  .ui-state-highlight {
+    background: $blue;
+    border-color: $blue;
+    color: $white;
+  }
+
+  .ui-state-active {
+    background: $lightBluishGrey;
+    border-color: $lightBluishGrey;
+    color: $white;
+  }
+}
+
+// ====================
+
+// JQUI timepicker
+.ui-timepicker-list {
+  z-index: 100000 !important;
+}
+
+//jQuery loupeAndLightbox Plugin
+.zooming-image-place {
+  .larger {
+    left: 0 !important;
+    bottom: 100% !important;
+  }
+}
+// ====================
+
+// reset styles to remove ui-lightness jquery ui theme from the tabs component (used in the add component problem tab menu)
+.ui-tabs {
+  padding: 0;
+  white-space: normal;
+}
+
+.ui-corner-all, .ui-corner-bottom, .ui-corner-left, ui-corner-top, .ui-corner-br, .ui-corner-right {
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+  border-top-right-radius: 0;
+  border-top-left-radius: 0;
+}
+
+
+.ui-widget-content {
+  border: 0;
+  background: none;
+}
+.ui-widget {
+  @extend %t-copy-base;
+}
+
+.ui-widget-header {
+  border:none;
+  background: none;
+}
+
+.ui-tabs .ui-tabs-nav {
+  padding: 0;
+}
+
+.ui-tabs .ui-tabs-nav li {
+  margin: 0;
+  padding: 0;
+  border: none;
+  top: 0;
+  margin: 0;
+  float: none;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.ui-tabs-nav {
+
+  li {
+    top: 0;
+    margin: 0;
+  }
+  a {
+    @extend %t-regular;
+    float: none;
+  }
+}
+
+.ui-tabs .ui-tabs-panel {
+  padding: 0;
+}
+
+// reapplying the tab styles from unit.scss after removing jquery ui ui-lightness styling
+.problem-type-tabs {
+  border:none;
+  list-style-type: none;
+  width: 100%;
+  @include linear-gradient(top, rgba(255, 255, 255, .4), rgba(255, 255, 255, 0));
+  //background-color: $lightBluishGrey;
+  box-shadow: 0 1px 0 rgba(255, 255, 255, 0.2) inset, 0 -1px 0 rgba(0, 0, 0, 0.2) inset;
+
+  li:first-child {
+    margin-left: $baseline;
+  }
+  li {
+    opacity: 0.8;
+
+    &:ui-state-active {
+      @extend %t-regular;
+      background-color: rgba(255, 255, 255, .3);
+      opacity: 1.0;
+    }
+    a:focus {
+      outline: none;
+      border: 0px;
+    }
+  }
+}
+
+.ui-autocomplete {
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: 0;
+  padding: 0;
+  cursor: default;
+  @include linear-gradient($gray-l5, $white);
+  border-right: 1px solid $gray-l2;
+  border-bottom: 1px solid $gray-l2;
+  border-left: 1px solid $gray-l2;
+  background-color: $gray-l5;
+  box-shadow: inset 0 1px 2px $shadow-l1;
+  color: $color-copy-emphasized;
+
+  li.ui-menu-item{
+    padding: 0;
+    margin: 0;
+
+    a {
+      color: $color-copy-emphasized;
+    }
+
+    a.ui-state-focus{
+      border: none;
+      background-color: $blue;
+      background: $blue;
+      color: $white;
+    }
+  }
+}
```

Comment:
Same here, no need to copy the file, add your overrides to _edraak-university-id.scss under the context of .university-id